tango_simlib/tango_sim_generator.py

```diff
@@ -504,6 +508,8 @@ def get_device_class(sim_data_files):
             parser_instance = get_parser_instance(data_file)
         elif extension in [".json"] and len(sim_data_files) < 2:
             parser_instance = get_parser_instance(data_file)
+        elif extension in [".fgo"] and len(sim_data_files) <2:
```

Comment:
You do not need another `elif..` block here. You can have the first `elif` block like this `..in [".fgo", ".json"] and ...`.